File: tests/fixtures/go/nested_control_flow.go
package fixtures

// Expected metrics:
// process: cognitive=TBD, cyclomatic=TBD, sloc=TBD
func process(items []int) int {
	total := 0
	if len(items) > 0 {
		for _, item := range items {
			if item > 0 {
				total += item
			}
		}
	}
	return total
}

File: tests/fixtures/cpp/nested_control_flow.cpp
// Expected metrics:
// process: cognitive=TBD, cyclomatic=TBD, sloc=TBD
int process(int* items, int len) {
    int total = 0;
    if (len > 0) {
        for (int i = 0; i < len; i++) {
            if (items[i] > 0) {
                total += items[i];
            }
        }
    }
    return total;
}
